clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -392,20 +393,18 @@ public class VenicePushJob implements AutoCloseable {
 
   /**
    * Config to enable single targeted region push mode in VPJ.
-   * In this mode, the VPJ will only push data to a single colo/region first, perform validation and finally
-   * leverage data recovery/repush to propagate data globally.
-   * The single solo/region is decided by the store config in {@link StoreResponse#getStore()}}.
+   * In this mode, the VPJ will only push data to a single region.
+   * The single region is decided by the store config in {@link StoreInfo#getNativeReplicationSourceFabric()}}.
    * For multiple targeted regions push, may use the advanced mode. See {@link #TARGETED_REGION_PUSH_LIST}.
    */
   public static final String TARGETED_REGION_PUSH_ENABLED = "targeted.region.push.enabled";
 
   /**
    * This is experimental config to specify a list of regions used for targeted region push in VPJ.
    * {@link #TARGETED_REGION_PUSH_ENABLED} has to be enabled to use this config.
-   * In this mode, the VPJ will only push data to the provided regions first, perform validation and finally
-   * leverage data recovery/repush to propagate data globally.
+   * In this mode, the VPJ will only push data to the provided regions.
    * The input should be split by comma, e.g. "dc-0,dc-1,dc-2".
```

Comment:
nit: The input is comma separated list of region names

Reply:
updated!